core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1302,7 +1304,7 @@ public void checkAccess(AlluxioURI path, CheckAccessContext context)
   }
 
   private void checkConsistencyRecursive(LockedInodePath inodePath,
-      List<AlluxioURI> inconsistentUris, boolean assertInconsistent)
+      List<AlluxioURI> inconsistentUris, boolean assertInconsistent, boolean syncMetadata)
```

Comment:
syncMetadata -> metadataSynced

Reply:
I was about to suggest the same thing before you did.

Reply:
done